### PR TITLE
Added an Anchor Tag in "see detail above"

### PR DESCRIPTION
**Pull request by AkashKumarSingh11032001**
I added an anchor tag in line number 39 and added a missing 'a' in line 72

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Let's keep it that way. Let's strive to be one of the friendliest communities in
 all of open source.
 
 As always, discuss Google Test in the official GoogleTest discussion group. You
-don't have to actually submit code in order to sign up. Your participation
+don't have to actually submit a code in order to sign up. Your participation
```
Comment (derekmauro) [inline CONTRIBUTING.md:72]: Can you revert this change? The added "a" is grammatically incorrect.

Comment (AkashKumarSingh11032001) [reply CONTRIBUTING.md:72]: ok
